blockchain/txSigner.go

```diff
@@ -45,7 +65,13 @@ func (ts *txSigner) GeneratePkBytes(skBytes []byte) ([]byte, error) {
 	return pk.ToByteArray()
 }
 
+func (xs *xSigner) serializeForSigning(msg []byte) ([]byte, error) {
+	msg = append(messagePrefix, msg...)
```

Comment:
I think you forgot about this: https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/main/src/signableMessage.ts#L46

Then - can we have a test that a signed message from js validates here? (plus verification as discussed in the call)

Reply:
done

Reply:
added also the test for that